Atomic/trust.py

```diff
@@ -182,9 +236,93 @@ def write_registry_config_file(self, reg_file, registry, sigstore):
 
     def get_sigstore_type_map(self, sigstore_type):
         """
-        Get the skopeo trust policy type for user-friendly value
+        Given the user-friendly trust type, get the skopeo value
         :param sigstore_type: one of web,local,atomic
         :return: skopeo trust policy type string
         """
-        t = { "web": "docker", "local": "dir", "atomic": "atomic" }
+        t = { "docker": "docker", "web": "docker", "local": "dir", "dir": "dir", "atomic": "atomic" }
+        if sigstore_type not in t:
+            raise ValueError("Invalid sigstore type %s" % sigstore_type)
         return t[sigstore_type]
+
+    def discover_sigstore(self, pull_image, confirm=None):
+        """
+        Check for registry/repo/sigstore metadata image
+        prompt user for trust on first use workflow
+        :param pull_image: image being pulled, used to discover matching sigstore meta image
+        :param confirm: string
+        """
+        if not util.get_atomic_config_item(['discover_sigstores'], util.get_atomic_config()):
```

Comment:
Actually can you just pass the assumeyes on the __init__ function.  And refer to it as assumeyes, so we have consistency.